srcCpp/perftest_publisher.cxx

```diff
@@ -14,6 +14,8 @@
 #include "CpuMonitor.h"
 #include "Infrastructure_common.h"
 
+/* A static variable must be initialize */
+unsigned int perftest_cpp::OVERHEAD_BYTES = 0;
```

Comment:
I think you have not considered how this will play with Micro, is it going to work? I think it will fail at compilation time.